scripts/create-missing-test-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2043
+for folder in tests; do
+	echo "./$folder"
+
+	FILE_NAMES=$(find ./$folder -name "*.go" | grep -v "_test.go" | grep -v "_moq.go" | grep -v "ginkgo" | sed 's/.go//g')
+
+	# if a filename_test.go does not exist, create it
+	for FILE_NAME in $FILE_NAMES; do
+		if [ ! -f "$FILE_NAME"_test.go ]; then
+			touch "$FILE_NAME"_test.go
```

Comment:
Why is the creation necessary at this stage? The file will be created later.

Reply:
Removed.